src/pheromones/include/pheromones/HiveConnectNetworkConfigGetRequestDTO.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PROPOLIS_HIVECONNECTNETWORKCONFIGGETREQUESTDTO_H
+#define PROPOLIS_HIVECONNECTNETWORKCONFIGGETREQUESTDTO_H
+
+#include "message.pb.h"
+
+class HiveConnectNetworkConfigGetRequestDTO {
+  public:
+    HiveConnectNetworkConfigGetRequestDTO() = default;
```

Comment:
Is the default constructor used anywhere?

Reply:
Yes, to send a simple get request to the HiveConnect